tests/jenkins/nrnivmodl.sh

```diff
@@ -6,7 +6,8 @@ set +x
 TEST_DIR="$1"
 
 . /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
-module load neuron/2018-10/python3/parallel intel
+module load intel
+module load `module av neuron 2>&1 | grep -o -m 1 'neuron.*/parallel' | awk -F' ' '{print $1}'`
```

Comment:
The script should check that the crazy `module av ...`  command actually finds something, and properly fails otherwise.
Also maybe use ^ $ in the regex to enforce it a bit.

Reply:
That's a good improvement :+1: 
I added a check if a neuron version is found. Hope that this is sufficient